src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -74,8 +74,6 @@ public abstract class AbstractTargetConfiguration implements TargetConfiguration
             "png", "jpg", "jpeg", "gif", "bmp", "ttf", "raw",
             "xml", "fxml", "css", "gls", "json", "dat",
             "license", "frag", "vert", "obj", "mtl", "js");
-    protected static final List<String> ENABLED_FEATURES = 
```

Comment:
Keep the feature as a static String, like `HOME_FINDER_FEATURE`, and add a comment like valid until GraalVM 21 or so.